### PR TITLE
Added favicon and title

### PR DESCRIPTION
**Pull request by Briansbum**
Little cosmetic tweak to make the app look better to users.

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <html>
 
 <head>
+  <title>sup3rS3cretMes5age</title>
+  <link rel="icon" tpye="image/png" href="/static/favicon.png">
```
Comment (rayrutjes) [inline static/index.html]: s/tpye/type/

Comment (Briansbum) [reply static/index.html]: Whoops, cad7cf4

**Comment by Haroenv** [comment]
Thanks for your contribution, we should add a favicon. I don't think that the size is suitable though, favicon usually is 64x64. If we're having a favicon, we should probably have it in a few sizes. I usually like to use [realfavicongenerator.net](https://realfavicongenerator.net)

Let's use an original design that works for this size

**Comment by eranchetz** [comment]
Thanks a lot for your contribution @Briansbum,
any chance you can update the favicon so I can merge this?

thanks again! 👍

**Comment by Briansbum** [comment]
@eranchetz icons have been added from the generator recommended by @Haroenv 

Sorry for the delay!

**Pull request merged**